collection-scripts/gather

```diff
@@ -38,6 +40,7 @@ elif [ "${reconcileStrategy}" = "standalone" ]; then
 else
     dbglog "Collecting entire ceph logs"
     gather_ceph_resources ${BASE_COLLECTION_PATH}
+    gather_odf_consumer ${BASE_COLLECTION_PATH}
```

Comment:
- there are scenarios where a single cluster will run in both provider & client modes and you'll be gathering only provider MG in that scenario.

Reply:
For such scenarios, the check for storagecluster does not work. We might need to change the if else logic conditions